packages/frontend/app/lib/message.server.ts

```diff
@@ -23,3 +23,24 @@ export const messageStorage = createCookieSessionStorage<
     secure: process.env.NODE_ENV === 'production'
   }
 })
+
+export function setMessage(session: Session, message: Message) {
```

Comment:
is this one used?

Reply:
Not currently, but I've kept it in case we will need to set a message without redirecting instantly.